QModManager/Utility/Dialog.cs

```diff
@@ -51,6 +51,12 @@ internal class Button
                     Process.Start(VersionCheck.bzNexus);
             });
 
+            internal static readonly Button Pirate = new Button("Close", () =>
+            {
+                Process.Start("https://www.youtube.com/watch?v=dQw4w9WgXcQ");
```

Comment:
Ha!

Reply:
That's intentional. It was supposed to be a prank lol

Reply:
Indeed, just laughing along with it xD